src/encoded/commands/write_submission_spreadsheets.py

```diff
@@ -850,6 +908,8 @@ def main():
     )
     parser.add_argument("--env", help="Environment", default="data")
     parser.add_argument("--item", help="Item name", nargs="+")
+    parser.add_argument("--tpc", help="TPC Submission items", action="store_true")
+    parser.add_argument("--gcc", help="GCC Submission items", action="store_true")
```

Comment:
They can't both be set, right? --gcc will be ignored then. Might be good to emit a warning if that happens. Not necessary though.